The pyproject.toml already sets the correct version, so the version detection
system doesn't even have to be replaced.
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -32,18 +32,12 @@
 poetry = "^1.6.1"
 
 [build-system]
-requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning>=1.0.0,<2.0.0"]
-build-backend = "poetry_dynamic_versioning.backend"
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
 
 [tool.poetry.scripts]
 auto-cpufreq = "auto_cpufreq.bin.auto_cpufreq:main"
 auto-cpufreq-gtk = "auto_cpufreq.bin.auto_cpufreq_gtk:main"
 
-# https://github.com/mtkennerly/poetry-dynamic-versioning
-[tool.poetry-dynamic-versioning]
-enable = true
-vcs = "git"
-format = "v{base}+{commit}"
-
 # SideNote
 # Regarding zip_safe = https://setuptools.pypa.io/en/latest/deprecated/zip_safe.html
